src/pages/Main/index.js

```diff
@@ -1,42 +1,91 @@
 import React from 'react';
 import Form from '../../components/Form';
-import List from '../../components/List';
+import ListAlta from '../../components/ListAlta';
+import ListMedia from '../../components/ListMedia';
+import ListBaixa from '../../components/ListBaixa';
 
 class Main extends React.Component {
   constructor(props){
     super(props);
 
     this.state = {
-      data: [],
+      data: [{"Alta": [], "Media": [], "Baixa": []}],
     }
   }
 
-  addTodo(event, val){
+  addTodo(event, val, prioridade){
     event.preventDefault();
+    
+    var todo = { text: val };
+    if (prioridade === "") {
+      prioridade = "Alta";
+    }
+
+    if (val === "") {
+      this.addTodo.stop()
```

Comment:
Vocês poderiam usar return, que teria o efeito parecido

Reply:
Foi ótimo por que o ``this.addTodo.stop()`` não funciona, dai erro